Week3/homework/2-step3.js

```diff
@@ -1,8 +1,30 @@
 'use strict';
-
+function sayThree() {
+  return 'sayThree';
+}
+function sayFive() {
+  return 'sayFive';
+}
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
-  // Add your code here
+  const temp = stopIndex - startIndex;
+  const outputArr = [];
+  for (let i = 0; i <= temp; i++) {
+    values[i] = startIndex;
+    startIndex++;
+  }
+  const results = values.filter(value => value % 3 === 0 || value % 5 ===0);
```

Comment:
actually you can use `values.forEach` immediately because you are checking there the equality again right?